tests/test_environment.py

```diff
@@ -0,0 +1,29 @@
+from dotenv import load_dotenv
+import os
+import sys
+
+load_dotenv()
+
+sys.path.append(os.path.abspath(''))
```

Comment:
Can you add a comment explaining why this is needed?

Reply:
Works without it, not sure why I added it.